src/Interfaces/MethodDescriptor.php

```diff
@@ -19,6 +19,11 @@ public function getParams();
      */
     public function getReturn();
 
+    /**
+     * @return boolean
+     */
+    public function isStatic();
+
     /**
      * @return Type[]
```

Comment:
ClassType again pls

Reply:
why would a method only be allowed to return a class?